config/versions.json

```diff
@@ -38,12 +38,24 @@
         "value": {
           "name": "Sitecore.NET 9.1.0 (rev. 001564)",
           "nugetVersion": "",
-          "vagrantBoxName": "w16s-sc91",
+          "vagrantBoxName": "w16s-sc910",
           "majorVersion": "9.1",
           "exactVersion": "9.1",
           "kernelVersion": "12.0.0",
           "netFrameworkVersion": "4.7.1"
         }
+      },
+      {
+        "name": "Sitecore.NET 9.1.1",
+        "value": {
+          "name": "Sitecore.NET 9.1.1",
+          "nugetVersion": "",
```

Comment:
Could you please search for all possible entries and drop this nugerVersion parameter everywhere